src/functionaltests/java/com/ericsson/ei/utils/MultiOutputStream.java

```diff
@@ -0,0 +1,49 @@
+package com.ericsson.ei.utils;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class MultiOutputStream extends OutputStream
+{
+    OutputStream[] outputStreams;
```

Comment:
I thinks should be private
and general comment about class move the "{" up in all your methods, should be after method not from the new line